friends/src/components/Friend.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import { Link } from "react-router-dom";
+
+export default class Friend extends Component {
+
+  constructor(props){
+    super(props);
+  }
```

Comment:
This constructor seems a bit unnecessary since you have no state (it'll pretty much already do this for you if you omit it).